ci/run.yml

```diff
@@ -0,0 +1,30 @@
+- hosts: controller
+  tasks:
+    - name: Install ansible so that ansible-playbook is available
+      ansible.builtin.pip:
+        name:
+          - ansible
+
+    - name: Set the repo dir
+      ansible.builtin.set_fact:
+        fvt_dir: "{{ ansible_env.HOME}}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}"
+
+    - name: Copy the callback_plugin to enable it
+      ansible.builtin.copy:
+        src: "{{ fvt_dir }}/callback_plugins/"
```

Comment:
This is failing.

This is because of the copy module. The `src` is on the ansible deployment host i.e. zuul executer or localhost.

What you're expecting is the equivalent of `cp fvt/callback_plugins fvt/ci/callback_plugins`

What's actually happening is closer to `scp zuul_executer:/fvt/callback_plugin controller:fvt/ci/callback_plugins`

You need to use the [`remote_src`](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/copy_module.html#parameter-remote_src) option to specify that the file is already present on the target host, and to copy from file_on_target to file_on_target.